src/components/base.css

```diff
@@ -1,25 +1,45 @@
 @import url('https://fonts.googleapis.com/css?family=Concert+One|Varela+Round|Oleo+Script');
 
-html {
-  background-color: #ececec;
+:root {
+  --background: #ececec;
+  --color: #333;
+  --muted: #999;
+  --blue: #80b3ff9c;
+  --purple: #A3A1DE;
+  --purple-dark: #9a98d4;
+  --vaiant1: #fff;
+  --vaiant2: #f4f4f4;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background: #272727;
+    --color: #fff;
+    --vaiant1: #3a3a3a;
```

Comment:
Also just gonna be nitpicky here, shall we rename "vaiant" to something else, more descriptive perhaps?

Reply:
I was trying to think of something more descriptive but could not find out what it should be 🙈 

`--vaiant2` is used for the gray-ish inputs and `--variant` is the background of heading input and the box around the status item. Do you have any suggestions for better names? 👼

Reply:
ah! sorry, maybe just `variant`? :D  then it makes sense

Reply:
agree, I'll make the changes :)